### PR TITLE
Update upgrade-nextgen-cd.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ For more information, go to [Harness FirstGen vs Harness NextGen](../../../getti
 
 The Harness FirstGen CD module will be phased out and will reach End of Life (EOL) in about 12 months. 
 
+### Timeline
+
```
Comment (michaelcretzman) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md:28]:
```suggestion
The FirstGen to NextGen CD upgrade will occur on the following dates:
```

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md [hunk 2]
```diff
@@ -24,6 +24,13 @@ For more information, go to [Harness FirstGen vs Harness NextGen](../../../getti
...
+1/1/2023 - maintenance mode. no new features, bugs and security issues will be fixed.
+10/1/2023 - end of SMP releases
+1/1/2024 - end of support
+4/1/2024 - SaaS product shutdown
```
Comment (michaelcretzman) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-nextgen-cd.md:32]:
```suggestion
- 1/1/2023. This is maintenance mode. No new features will be added and no new bugs and security issues will be fixed.
- 10/1/2023. End of SMP releases.
- 1/1/2024. End of support for FirstGen CD.
- 4/1/2024. FirstGen SaaS product shutdown.
```

**Review by michaelcretzman — CHANGES_REQUESTED** [review]
Please input my suggestions before you merge.

**Comment by thisrohangupta** [comment]
Accepted the Suggestions @michaelcretzman

**Review by thisrohangupta — COMMENTED** [review]
Date Suggestion accepted.

**Review by michaelcretzman — APPROVED** [review]

**Pull request merged**